scripts/build_lib.sh

```diff
@@ -4,6 +4,10 @@ source ./scripts/test_lib.sh
 
 GIT_SHA=$(git rev-parse --short HEAD || echo "GitNotFound")
 VERSION_SYMBOL="${ROOT_MODULE}/api/v3/version.GitSHA"
+GO_LDFLAGS="${GO_LDFLAGS:-"-s -w"}"
+GOOS="${GOOS:-"linux"}"
+GOARCH="${GOARCH:-"amd64"}"
+BINDIR="${BINDIR:-"bin"}"
```

Comment:
Requires default value to pass unbound variable shell check.

Reply:
This is inconsistent with proposal in https://github.com/etcd-io/etcd/pull/15519
cc @fuweid 
Could you align on this to avoid conflicts?

Reply:
Yeah, I will rebase the commit on the referenced PR.